flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemCommitterTest.java

```diff
@@ -40,12 +44,24 @@ class FileSystemCommitterTest {
     private FileSystemFactory fileSystemFactory = FileSystem::get;
 
     private TableMetaStoreFactory metaStoreFactory;
+    private List<PartitionCommitPolicy> policies;
+    private ObjectIdentifier identifier;
     @TempDir private java.nio.file.Path outputPath;
     @TempDir private java.nio.file.Path path;
 
     @BeforeEach
     public void before() throws IOException {
         metaStoreFactory = new TestMetaStoreFactory(new Path(outputPath.toString()));
+        policies =
+                PartitionCommitPolicy.createPolicyChain(
+                        Thread.currentThread().getContextClassLoader(),
+                        "metastore,success-file",
+                        null,
+                        "_SUCCESS",
+                        () -> {
+                            return new LocalFileSystem();
```

Comment:
Can be `LocalFileSystem::new`

Reply:
This will fail to compile because `FileSystem is not a functional interface`.